ports/geckolib/gecko_bindings/tools/regen.py

```diff
@@ -0,0 +1,366 @@
+#!/usr/bin/env python
```

Comment:
Whoops... Thanks!

(BTW I meant to mention you above, not @jgraham, who uses to be busy and not too much around servo)